src/slam/examples/internal-sample.js

```diff
@@ -70,8 +70,8 @@ function instanceHandler(slamInstance) {
     console.log('error:' + e);
   });
 
-  function saveOccupancyMapAsPpm() {
-    slamInstance.saveOccupancyMapAsPpm( __dirname + '/map.ppm', false)
+  function saveOccupancyMap() {
+    slamInstance.saveOccupancyMap( __dirname + '/map.ppm', false)
```

Comment:
Please create another test file to test the new API. Or you can create the test case in the test folder.